database/migrations/2017_12_20_173343_add_foreign_columns_to_users_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddForeignColumnsToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->integer('team_id')->unsigned()->nullable();
+            $table->integer('role_id')->unsigned()->nullable();
```

Comment:
Not nullable.

Reply:
^

Reply:
Both or just the role_id?

Reply:
Just the `role_id`, because every user has a role from the beginning.